src/MyPlot/MyPlot.php

```diff
@@ -414,14 +414,7 @@ public function getPlotMid(Plot $plot) {
 
 		$plotSize = $plotLevel->plotSize;
 		$pos = $this->getPlotPosition($plot);
-		if($plot->X >= 0 and $plot->Z >= 0)
-			$pos->add(floor($plotSize / 2), 1, floor($plotSize / 2));
-		if($plot->X < 0 and $plot->Z > 0)
-			$pos->add(-floor($plotSize / 2), 1, floor($plotSize / 2));
-		if($plot->X > 0 and $plot->Z < 0)
-			$pos->add(floor($plotSize / 2), 1, -floor($plotSize / 2));
-		if($plot->X < 0 and $plot->Z < 0)
-			$pos->add(-floor($plotSize / 2), 1, -floor($plotSize / 2));
+		$pos = new Position($pos->getFloorX() + ($plotSize / 2) + 0.5, $pos->getFloorY() + 1, $pos->getFloorZ() + ($plotSize / 2) + 0.5);
```

Comment:
Are you certain that this returns the exact center every time? It doesn't really account for positive or negative x/z coords

Reply:
I'm testing now

Reply:
Have you tested this?

Reply:
Would i create untested PR's? ;) Sure i did. Was my explaination above understandable? I wanted to say that: the plotpos is always the lowest x and z of a plots area. in order to get the middle, you would always have to go to the positive direction. Also, currently it just teleports to the plotpos because the position doesn't get changed in the code anyways